src/Table.luna

```diff
@@ -71,27 +85,128 @@ class Table:
     def schema:
         wrapper = self.ptr.schema
         SchemaVal wrapper
+
+    # Returns a single `Column` value from `Table` by integer index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > column = table.columnAt 1
+    #
+    # `index`: Integer column index.
+    # `return`: `Column` value from the `Table` for given index.
+
     def columnAt index:
         columnWrapper = self.ptr.columnAt index
         Column.fromColumnWrapper columnWrapper
+    
+    # Insert a single `Column` value to `Table` under given index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > l3 = [21,22,23,24,25]
+    # > col3 = Column.fromList "col3" Int64Type l3 
+    # > newTable = table.addColumn 1 col3
+    #
+    # `index`: Integer index for newly inserted column.
+    # `column`: `Column` value to insert.
+    # `return`: `Table` value containing previous `table` with newly inserted column.
+
     def addColumn index column:
         self.fromWrapper $ self.ptr.addColumn index column.ptr
     def concat ts:
         cols = ts . each .toList . concat
         self.fromColumns cols
     def + that: Table.concat [self, that]
+
+    # Returns a `Table` value contain column matching given name.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > column = table.at "col1"
+    #
+    # `name`: Label for column.
+    # `return`: `Table` value from the `table` for given name.
+
     def at name:
         self.columns [name]
+    
+    # Add a single `Column` value to `Table` with given name.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > l3 = [21,22,23,24,25]
+    # > col3 = Column.fromList "col3" Int64Type l3 
+    # > newTable = table.setAt "foo" col3
+    #
+    # `name`: String name for newly added column.
```

Comment:
What if there is a column with that name in a table? Does column order matter? If yes, what is the algorithm for determining where to put the new column?

Reply:
new column is always at the end of the table

Reply:
That's worth adding to the documentation ;)

Reply:
Why "String name"? There are no other kinds of names. I'd say just "Name".
Or, if we want to include type information, it should be clear what is the typename (Text).